src/SkynetServer.Web/Controllers/MailConfirmationController.cs

```diff
@@ -38,6 +39,11 @@ public IActionResult Post(string token)
                 return View("Invalid");
             if (confirmation.ConfirmationTime == default(DateTime))
             {
+                // Remove confirmations that have become obsolete due to an address change
+                // TODO: Add protocol interaction to inform clients about a suceeded address change
+                _database.MailConfirmations.RemoveRange(
```

Comment:
Why do we use `_database` here and not `database`? In all other classes of  the server, private fields are without the underscore.